### ISSUE TITLE
Accesibility insights quick pass issues.

### ISSUE THREAD

**Issue by jeffj6123** [Microsoft/service-fabric-explorer#184]
**Accessibility Compliance: Images must have alternate text**

Instances of missing alt texts.
<img ng-if="badgeClass" class="badge-icon ng-scope" ng-src="images/badge-ok.svg" src="images/badge-ok.svg">
div[badge-class="\'badge-ok\'"] > .badge-icon[ng-src$="badge-ok.svg"][src$="badge-ok.svg"]

<img ng-if="badgeClass" class="badge-icon ng-scope" ng-src="images/badge-warning.svg" src="images/badge-warning.svg">
img[ng-src$="badge-warning.svg"]

<img ng-if="badgeClass" class="badge-icon ng-scope" ng-src="images/badge-error.svg" src="images/badge-error.svg">
img[ng-src$="badge-error.svg"]

<img ng-if="badgeClass" class="badge-icon ng-scope" ng-src="images/badge-ok.svg" src="images/badge-ok.svg">
div[text="systemApp\.healthState\.text"] > .badge-icon[ng-src$="badge-ok.svg"][src$="badge-ok.svg"]

**Accessibility Compliance: Form elements must have labels**

<input type="search" class="input-flat ng-pristine ng-untouched ng-valid ng-empty" autocomplete="off" placeholder="Search Cluster" ng-model="model" ng-model-options="{ debounce: 300 }" aria-invalid="false">
.ng-empty[type="search"][placeholder="Search\ Cluster"]

<input type="search" class="input-flat ng-pristine ng-untouched ng-valid ng-empty" autocomplete="off" placeholder="Search list" ng-model="model" ng-model-options="{ debounce: 300 }" aria-invalid="false">
.ng-empty[type="search"][placeholder="Search\ list"]

**Accessibility Compliance: <html> element must have a lang attribute**

**Accessibility Compliance: duplicate-id: id attribute value must be unique**
<script type="text/ng-template" id="nodeTemplate">
.node[aria-label="Applications"][sfx-tree-scroll-to=""] > .children[node="child\.childGroupViewModel"][ng-show="node\.isExpanded"] > script[type="text\/ng-template"]

**Accessibility Compliance: Required ARIA attributes must be provided**

<sfx-slider min-label-id="slider-min-label" max-label-id="slider-max-label" class="slider ui-slider ui-corner-all ui-slider-horizontal ui-widget ui-widget-content" title="Change how often content is updated by clicking or using arrow keys" aria-label="Change how often content is updated" role="slider">sfx-slider
--
